src/content/doc-surrealql/statements/define/config.mdx

```diff
@@ -0,0 +1,373 @@
+
+
+> [!WARNING]
+> By running SurrealDB with the GraphQL module enabled, you are opting into an experimental feature. While the GraphQL module is fully functional, it is still considered experimental and may not be as stable as the core SurrealQL module which means we cannot guarantee that it will provide the same security guarantees. It is not recommended for production use. We welcome your feedback and contributions to help improve the feature and make it more robust.
+
+## Requirements
+
+- You **must** be authenticated as a **root**, **namespace**, or **database** user before you can use the `DEFINE CONFIG GRAPHQL` statement.
+- You **must** select your **namespace** and **database** before you can use the `DEFINE CONFIG GRAPHQL` statement.
+- You **must** define at least one table in your database for the GraphQL API to function.
+- You **must** have started the SurrealDB instance with the `SURREAL_EXPERIMENTAL_GRAPHQL` environment variable set to `true`.
+
+## Statement syntax
+
+```surql title="SurrealQL Syntax"
+
+DEFINE CONFIG [OVERWRITE | IF NOT EXISTS] GRAPHQL [ AUTO | NONE] 
+  [ TABLES (AUTO | NONE | INCLUDE table1, table2, ...) ] 
+  [ FUNCTIONS (AUTO | NONE | INCLUDE [function1, function2, ...] | EXCLUDE [function1, function2, ...]) ] 
+  [ COMMENT @string ]
+```
+
+
+## Important Notes
+
+- The `DEFINE CONFIG GRAPHQL` statement **must** be executed before any GraphQL queries can be made.
+- If you attempt to use the GraphQL API without defining the configuration, you will receive a `NotConfigured` error.
+- If no tables are defined in the database, you will receive an error stating "No tables found in database" when attempting to use the GraphQL API.
+
+## Example usage
+
+```surql
+-- Define GraphQL configuration 
+DEFINE CONFIG GRAPHQL  AUTO;
+```
+
+## Tables Configuration
+
+The `TABLES` clause in the `DEFINE CONFIG GRAPHQL` statement specifies how tables are exposed via GraphQL. There are four options for the `TABLES` configuration:
+
+- `AUTO`: Automatically include all tables in the GraphQL schema.
+- `NONE`: Do not include any tables in the GraphQL schema.
+- `INCLUDE`: Specify a list of tables to include in the GraphQL schema.
+- `EXCLUDE`: Specify a list of tables to exclude from the GraphQL schema.
+
+### `AUTO`
+
+When you specify `TABLES AUTO`, all tables in the database are automatically included in the GraphQL schema.
+
+```surql
+DEFINE CONFIG GRAPHQL TABLES AUTO;
+```
+
+### `NONE`
+
+When you specify `TABLES NONE`, no tables are included in the GraphQL schema.
+
+```surql
+DEFINE CONFIG GRAPHQL TABLES NONE;
+```
+
+### `INCLUDE`
+
+You can specify a list of tables to include in the GraphQL schema using the `INCLUDE` clause. The list of tables is specified as a comma-separated list without brackets.
+
+```surql
+DEFINE CONFIG GRAPHQL TABLES INCLUDE user, post, comment;
+```
+
+> [!NOTE]
+> The `EXCLUDE` option for `TABLES` is currently not implemented.
+
+## Functions Configuration
```

Comment:
should note here that the statement works, but doesn't do anything as functions aren't supported yet